### DISCUSSION TITLE
How to handle MGMT for CCS-710P in campus design.

### DISCUSSION THREAD

**Discussion by morinpa** [aristanetworks/avd#3104] [Q&A] [answered]
I am trying to do a campus L2LS design that contains some 710P switches.  These do not have a Management interface, but the designs are pretty set in their ways to have one and a MGMT vrf also.  

I can kind of work around it with structured configs and moving things around, but it is not pretty.  

Is there a correct way to handle this?

**Comment by ClausHolbechArista** [comment]
We have added some inband management knobs for use cases like this:
https://avd.arista.com/4.2/roles/eos_designs/docs/input-variables.html#node-type-inband-management

Notice you can _either_ use `inband_mgmt_subnet` _or_ `inband_mgmt_ip` depending on your needs. Please let us know if you feel like something is missing.

**Reply by morinpa** [reply]
Hi,  I have used this, but the rest of the mgmt configs that come with the interface is still being rendered.  Like the default route in the mgmt VRF and the MGMT vrf being configured for the name servers also even it the Management0 interface is not being configured on a specific switch.  Am I missing something?

**Reply by ClausHolbechArista** [reply, marked as answer]
Each of these are caused by different issues:
- The default route in the MGMT VRF comes from the `mgmt_gateway` variable, so avoid setting that.
- The name server VRF is currently hardcoded to take the out of band VRF. You could set `mgmt_interface_vrf: "default"` to work around this. I am working on fixing all management protocols to make them work better with both out-of-band and inband management.
- See the new `source_interfaces` released in 4.3.0. It is not complete yet, but it will show the direction.
